core/src/main/java/io/gnosis/kouban/core/utils/DateUtils.kt

```diff
@@ -0,0 +1,51 @@
+package io.gnosis.kouban.core.utils
+
+import android.app.DatePickerDialog
+import android.content.Context
+import android.widget.DatePicker
+import androidx.core.os.ConfigurationCompat
+import java.text.SimpleDateFormat
+import java.util.*
+
+fun Long.asFormattedDateTime(context: Context, dateFormat: String = DateFormats.default): String =
+    Date().apply { time = this@asFormattedDateTime * 1000 }.let { date ->
+        with(SimpleDateFormat(dateFormat, ConfigurationCompat.getLocales(context.resources.configuration)[0])) {
+            format(date)
+        }
+    }
+
+fun Long.asFormattedDateTime(formatter: SimpleDateFormat): String =
+    Date().apply { time = this@asFormattedDateTime * 1000 }.let { date -> formatter.format(date) }
+
+fun showDatePickerDialog(context: Context, date: Date? = null, onDatePicked: (Date) -> Unit) {
+    val datePickedListener = DatePickerDialog.OnDateSetListener { _, year, month, dayOfMonth ->
+        val pickedDate = Calendar.getInstance().apply {
+            set(Calendar.YEAR, year)
+            set(Calendar.MONTH, month)
+            set(Calendar.DAY_OF_MONTH, dayOfMonth)
+            set(Calendar.HOUR_OF_DAY, 0)
+            set(Calendar.MINUTE, 0)
+            set(Calendar.SECOND, 0)
+            set(Calendar.MILLISECOND, 0)
+        }
+        onDatePicked(pickedDate.time)
+    }
+    val startDate = Calendar.getInstance().apply { date?.let { time = date } }
+    with(startDate) {
+        DatePickerDialog(
```

Comment:
date picker dialog should be styled using app primary

Reply:
It's using the default app theme. I think it is using it. How should I change it to what you are saying? I am not sure I understand what you mean

Reply:
It's using the default app theme. I think it is using it. How should I change it to what you are saying? I am not sure I understand what you mean

Reply:
It's using the default app theme. I think it is using it. How should I change it to what you are saying? I am not sure I understand what you mean

Reply:
![image](https://user-images.githubusercontent.com/17750984/76449474-6fe55480-63cc-11ea-9c07-f466cf38e985.png)
<!-- Theme.AppCompat.Light.Dialog -->
<style name="DialogTheme" parent="Theme.AppCompat.Light.Dialog">
    <item name="colorAccent">@color/primary</item>
</style>
DatePickerDialog(context,R.style.DialogTheme,...

Reply:
this header color of the dialog should be changed

Reply:
No problem man, I was just a bit clueless regarding what you were requesting 😅 of course I can change it :)

Reply:
I thought it was something specifically wrong with that line